src/Microsoft.ML.OnnxTransform/OnnxUtils.cs

```diff
@@ -344,6 +344,7 @@ public static void CopyTo<T>(Tensor tensor, T[] dst)
             {
                 var typedDst = (System.Single[])(object)dst;
                 tensor.CopyTo(typedDst);
```

Comment:
❔  Can you link to the definition of `CopyTo`?

Reply:
Sorry for the delay. The link to that function is available at below. Please add any comments you may have.

https://aiinfra.visualstudio.com/Sonoma/Sonoma%20Team/_git/Sonoma?path=%2Fmlscoring%2Fserving%2Fmanaged%2Flibrary_netstandard%2FTensorTemplate.cs&version=GBmaster